analytics/lib/counts.py

```diff
@@ -58,11 +75,15 @@ def process_count_stat(stat, fill_to_time):
 
```

Comment:
Any change to FillState should be logged, e.g. the first two branches of the conditional above. 
INITIALIZE property end_time
UNDO START property end_time
UNDO DONE property 3ms/5r
are reasonable log lines.